src/promptflow-evals/promptflow/evals/_common/constants.py

```diff
@@ -31,13 +31,33 @@ class Tasks:
     """Defines types of annotation tasks supported by RAI Service."""
 
     CONTENT_HARM = "content harm"
+    PROTECTED_MATERIAL = "protected material"
+
+
+class _InternalTasks:
+    """Defines types of annotation tasks that are supported for internal use. Such tasks are
+    experimental and subject to potential change or migration to the main Evaluation Metrics
+    enum over time."""
+
+    ECI = "eci"
 
 
 class EvaluationMetrics:
-    """Evaluation metrics for content safety evaluation."""
+    """Evaluation metrics to aid the RAI service in determining what
+    metrics to request, and how to present them back to the user."""
```

Comment:
Where does the "how to present them back to the user" part come in?

Reply:
Different metrics have slightly different output formats. This is used a bit in determining which to choose.